scriptmodules/system.sh

```diff
@@ -181,18 +194,30 @@ function platform_rpi2() {
     __default_cflags="-O2 -mcpu=cortex-a7 -mfpu=neon-vfpv4 -mfloat-abi=hard"
     __default_asflags=""
     __default_makeflags="-j2"
-    __platform_flags="arm armv7 rpi"
+    __platform_flags="arm armv7 neon rpi"
     # there is no support in qemu for cortex-a7 it seems, but it does have cortex-a15 which is architecturally
     # aligned with the a7, and allows the a7 targetted code to be run in a chroot/emulated environment
     __qemu_cpu=cortex-a15
     __has_binaries=1
 }
 
+function platform_rpi3() {
+    platform_rpi2
+}
+
+function platform_rpi3-64() {
+    __default_cflags="-O2 -mcpu=cortex-a53 -mfpu=neon-vfpv4 -mfloat-abi=hard"
```

Comment:
Ok. I have not find any info if -mfpu=neon-fp-armv8  is 64Bit exclusive. Will do some testing if the new pi arrives.